src/main/java/jdbchomework/console/MenuLauncher.java

```diff
@@ -13,5 +18,14 @@ public MenuLauncher(MainMenu mainMenu, MenuContents menuContents) {
     public void consoleMain() {
         menuContents.makeMenus();
         mainMenu.menu();
+        try {
+            reader.close();
+        } catch (IOException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)